narwhal/consensus/src/bullshark.rs

```diff
@@ -236,6 +237,21 @@ impl Bullshark {
         }
     }
 
+    // Returns the PublicKey of the authority which is the leader for the provided `round`.
+    // Pay attention that this method will return always the first authority as the leader
+    // when used under a test environment.
+    pub fn leader_authority(committee: &Committee, _round: Round) -> PublicKey {
```

Comment:
nit: remove underscore of round

Reply:
Ah actually I had to do that because linter was complaining about `round`  not being used (I guess because the `cfg_if::cfg_if!` macro which enables different paths).